earthpy/tests/test_spatial.py

```diff
@@ -37,3 +40,100 @@ def test_extent_to_json():
 
     with pytest.raises(AssertionError):
         es.extent_to_json([0, 1, 1, 0])
+
+
+# Create points GDF
+pts = np.array([[2, 2],
+                [3, 4],
+                [9, 8],
+                [-12, -15]])
+
+locs = [Point(xy) for xy in pts]
+locs_gdf = gpd.GeoDataFrame(locs,
+                            columns=['geometry'],
+                            crs={'init': 'epsg:4326'})
+
+# TODO? Create a create gdf from list function since we do it like 3
+# times in this test file already
+
+
+############## LINE OBJECTS #######
+
+linea = LineString([(1, 1), (2, 2), (3, 2), (5, 3)])
+lineb = LineString([(3, 4), (5, 7), (12, 2), (10, 5), (17, 17)])
+
+linez_gdf = gpd.GeoDataFrame([1, 2],
+                             geometry=[linea, lineb],
+                             crs={'init': 'epsg:4326'})
+
+############## POLY OBJECTS ################
+
+# Create crop box
+poly_inters = Polygon([(0, 0), (0, 10), (10, 10), (10, 0), (0, 0)])
+poly_in_gdf = gpd.GeoDataFrame([1],
+                             geometry=[poly_inters],
+                             crs={'init': 'epsg:4326'})
+
+# Shift by 20
+poly_out_gdf = poly_in_gdf.copy()
+poly_out_gdf = poly_in_gdf.geometry.apply(lambda x: shapely.affinity.
+                                          translate(x, xoff=20, yoff=20))
+# Buffer points to create multi poly
+locs_buff = locs_gdf.copy()
+locs_buff["geometry"] = locs_gdf.buffer(4)
+
+# Create geom w "donut hole"
+donut_geom = gpd.overlay(locs_buff, poly_in_gdf,
+                         how='symmetric_difference')
+
+multi_poly = donut_geom.unary_union
+multi_gdf = gpd.GeoDataFrame(gpd.GeoSeries(multi_poly),
+                   crs={'init': 'epsg:4326'})
+multi_gdf = multi_gdf.rename(columns={0: 'geometry'}).\
+    set_geometry('geometry')
+
+# LINE OBJECTS
```

Comment:
?

Reply:
These comments were me trying to sort out what sets of functions we might consider for gdf creation. I created points, lines and polys and wants to keep the creation of each separate in the code in hopes it would be easy to identify how it could be better modularized! was hoping you'd see that and maybe address whether we should consider functions to create gdfs.

Reply:
This one in particular seemed weird, since I couldn't find any line objects below it!

Reply:
We could totally write functions to generate these geodataframes, especially if we need to do it elsewhere.

Reply:
ok cool. i'll work on cleaning up the code first with smaller expressive functions. Then maybe we create functions for GDF creation.